infra/azure/__main__.py

```diff
@@ -161,17 +161,32 @@ def get_kubeconfig(
     ),
 )
 
+# Patch default namespace with pod security policies
+default_ns = Namespace(
+    "default-ns",
+    metadata=ObjectMetaArgs(
+        name="default",
+        labels={"pod-security.kubernetes.io/enforce": "restricted"},
```

Comment:
Because we use this in a number of places, it would be good to make it a variable.

We can always use `{...} | d` if we need to merge labels.